assets/templates/cards-component.html

```diff
@@ -71,32 +71,54 @@
         <div class="dp-align-center">
           <div>
             <h3 class="dpsg-tit-card m-l-30">Card Ribbon left-top</h3>
-            <div class="dp-card-ribbon-lt" data-ribbon="Recomendado"></div>
+            <div class="dp-card dp-recommended">
```

Comment:
What do you think about _highlighthed_ in place of _recommended_?

```suggestion
            <div class="dp-card dp-highlighthed">
```